.rubocop.yml

```diff
@@ -137,9 +137,8 @@ Style/FloatDivision:
 Style/FrozenStringLiteralComment:
   Enabled: false
 
-# Allow use of 1.8 and 1.9 style hashes, but not both in the same hash
 Style/HashSyntax:
-  EnforcedStyle: no_mixed_keys
+  EnforcedStyle: ruby19
```

Comment:
[Docs](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/HashSyntax)

Should we use `ruby19_no_mixed_keys` style instead?

<img width="458" alt="Screenshot 2024-12-20 at 11 36 45 PM" src="https://github.com/user-attachments/assets/34ae5bcf-151a-4164-8d67-c6e2b0b2f57c" />

vs

<img width="539" alt="Screenshot 2024-12-20 at 11 37 02 PM" src="https://github.com/user-attachments/assets/fc8ce427-8db5-46e2-bc0c-bd20f6dc9a5b" />

Reply:
Related PR: https://github.com/Root-App/root-monorepo/pull/65149

@mwise, why was `ruby19_no_mixed_keys` chosen for the claims system?